.github/workflows/release.yml

```diff
@@ -96,6 +97,6 @@ jobs:
       env:
```

Comment:
You can modernize this workflow even further by,
1. removing this `env` parameter
2. adding these two lines right below the `runs-on` parameter of the `publish_release` job
```yaml
    permissions:
      contents: write  # Required by 'action-gh-release'
```

Reply:
Done in https://github.com/adobe-type-tools/pyots/pull/25/commits/108f86650de3195e580b5bbd7fc23c9522da7f01